bin/dfx-network-stop

```diff
@@ -17,9 +17,10 @@ if [[ "$DFX_NETWORK" == "local" ]]; then
   echo "Waiting for replica to stop..."
   for ((i = 100; i > 0; i--)); do
     printf '\r % 4d' "$i"
-    pgrep replica || break
+    pgrep replica 2>/dev/null || break
```

Comment:
Printing the replica pid is incompatible with the progress printout that is trying to reuse the line.